e2e_tests/tests/config.py

```diff
@@ -14,9 +14,9 @@
 
 
 DEFAULT_TF1_CPU_IMAGE = "determinedai/environments:py-3.7-pytorch-1.7-tf-1.15-cpu-6218891"
-DEFAULT_TF2_CPU_IMAGE = "determinedai/environments:py-3.8-pytorch-1.12-tf-2.8-cpu-6218891"
+DEFAULT_TF2_CPU_IMAGE = "determinedai/environments-dev:py-3.8-pytorch-1.12-tf-2.12-cpu-0.21.2"
```

Comment:
Why is the temporary use of `determinedai/environments-dev` useful? Is there a concern about publishing a 2.12-tagged image to `determinedai/environments`?

Reply:
It's just because my PR that's creating these images is still open: https://github.com/determined-ai/environments/pull/196

Reply:
Will you commit the other before committing this one? If so, then could we amend this PR to have the final path?

Reply:
This PR and the environments PR can be merged in any order. Image names such as these will be changed in a bumpenv PR after both of the aforementioned are merged. Any necessary changes to the CircleCI workflows will also be done in that bumpenv PR.

Reply:
> If so, then could we amend this PR to have the final path?

I hadn't been sure about this part of the rollout plan. I see you've done exactly this. Thanks!